addon/styles/_frost-autocomplete.scss

```diff
@@ -0,0 +1,95 @@
+$frost-input-select-option-row-height: 30px;
+
+.frost-autocomplete {
+  .frost-select {
+    border: 0;
+  }
+
+  .frost-text {
+    width: 100%;
+    height: $frost-input-select-option-row-height;
+  }
+
+  .frost-text-clear > svg {
+    top: 2px;
+    left: 2px;
+  }
+}
+
+.frost-autocomplete-loading {
+  display: block;
+  height: 80px;
+  margin: auto;
+  margin-top: 25px;
+
+  svg {
+    height: 50px;
+  }
+}
+
+// sass-lint:disable force-element-nesting
```

Comment:
There's not a way to construct this without needing to use `sass-lint:disable force-element-nesting`?  This is new code - we should not be employing overrides.